prime-router/metadata/fhir_mapping/hl7/datatypes/EI/Organization.yml

```diff
@@ -0,0 +1,46 @@
+resourceType: Organization
+
+id:
+  type: STRING
+  valueOf: "GeneralUtils.generateResourceId()"
+  expressionType: JEXL
+
+identifier:
+  expressionType: nested
+  generateList: true
+  expressions:
+    - expressionType: nested
+      expressionsMap:
+        value:
+          type: STRING
+          expressionType: HL7Spec
+          valueOf: EI.2
+        extension:
+          generateList: true
+          expressionType: nested
+          expressionsMap:
+            url:
+              type: SYSTEM_URL
+              value: identifier-namespace-id
+            valueBoolean:
+              valueOf: "true"
+    - expressionType: nested
+      expressionsMap:
+        value:
+          type: STRING
+          expressionType: HL7Spec
+          valueOf: EI.3
+        type:
+          expressionType: resource
+          vars:
+            code: EI.4
+          valueOf: datatype/CodeableConcept_var
```

Comment:
There are a number of these mappings directed at the old schema files. Should these use the new CWE mappings?

Reply:
So EI.4 is not a CWE actually.  This is a case where we are taking just string and inserting into a coding.code structure.  I've been using this existing one (from the library) for creating that structure since it's convenient.

Based on @brick-green topic at standup today though, I could see a good argument for creating an `ID -> CodeableConcept` mapping, so maybe I'll just do that now